build/container/Makefile

```diff
@@ -1,12 +1,31 @@
 .DEFAULT_GOAL := build/base
 
-INFRASTRUCTURE_WORKSPACE       ?= workspace
-PROJECT_WORKSPACE              ?= $(CURDIR)/$(INFRASTRUCTURE_WORKSPACE)
-
-TARGET_DIR						= $(PROJECT_WORKSPACE)/target
-NRI_INTEGRATIONS_FILE			= $(CURDIR)/../embed/integrations.version
-NRI_PKG_DIR						?= ${PROJECT_WORKSPACE}/${BASE_OHIS}
-AGENT_BUILD_TARGET_DIR			?= $(CURDIR)/../../target/bin/linux_amd64
+INFRASTRUCTURE_WORKSPACE	?= workspace
+PROJECT_WORKSPACE		?= $(CURDIR)/$(INFRASTRUCTURE_WORKSPACE)
+
+TARGET_DIR			?= $(PROJECT_WORKSPACE)/target/$(GOOS)_$(DOCKER_ARCH)
+NRI_INTEGRATIONS_FILE		= $(CURDIR)/../embed/integrations.version
+NRI_PKG_DIR			?= ${PROJECT_WORKSPACE}/${BASE_OHIS}
+AGENT_BUILD_TARGET_DIR		?= $(CURDIR)/../../dist
+GOOS				?= linux
+DOCKER_ARCH				?= amd64
+DOCKER_BUILDX_CMD		?=
+DOCKER_BUILDX_ARGS		?= --platform $(GOOS)/$(DOCKER_ARCH)
```

Comment:
This only allows to specify one platform at once, which won't in turn allow to create multiarch images. For the latter, one usually does:
`docker buildx build --platform linux/amd64,linux/arm64`

Reply:
Due to the OHIs this is not gonna be possible. I've worked around this in another commit.